google/cloud/aiplatform/metadata/schema/google/artifact_schema.py

```diff
@@ -323,12 +357,34 @@ def __init__(
             check the validity of state transitions.
         """
         extended_metadata = copy.deepcopy(metadata) if metadata else {}
-        if au_prc:
+        if aggregation_type:
+            if aggregation_type not in _CLASSIFICATION_METRICS_AGGREGATION_TYPE:
```

Comment:
Could you add a test to validate this error scenario.

Reply:
Added Todo for this